tests/specs/profile/profile.spec.ts

```diff
@@ -50,19 +61,30 @@ test.describe('Gaia request', () => {
     await onboardingPage.signInWithTestAccount(extensionId);
     testAppPage = await TestAppPage.openDemoPage(context);
     await testAppPage.signIn();
+  });
+
+  test('should send a signed profile token to gaia', async ({ context }) => {
     const accountsPage = await context.waitForEvent('page');
     await accountsPage.locator('text="Account 2"').click({ force: true });
     await testAppPage.page.bringToFront();
-    await testAppPage.page.click('text=Profile');
+    await testAppPage.page.click('text=Profile', {
+      timeout: 30000,
+    });
     await accountsPage.close();
-  });
 
-  test.skip('should send a signed profile token to gaia', async ({ context }) => {
     await testAppPage.clickUpdateProfileButton();
     const profileUpdatingPage = new UpdateProfileRequestPage(await context.waitForEvent('page'));
 
     const requestPromise = profileUpdatingPage.page.waitForRequest('https://hub.blockstack.org/*');
```

Comment:
Does this trigger at all? I half-looked into this and couldn't get this to log

Reply:
I did see the log when using `--ui` flag running tests in UI mode.